update profiles, qualities, variants... of TiZYX printers

TiZYX opened the pull request:
Hello,
We updated the profiles, qualities and variants of all TiZYX printers.
Thank you for your time
Yours faithfully

TiZYX commented:
This is linked with [https://github.com/Ultimaker/fdm_materials/pull/86](url)

TiZYX commented:
I removed layer_options from non-global profiles.

For the speeds, I want to have a default speed (currently in global profiles) and just change those speeds for the flex material. That's why I modify the speeds in the non-global quality profiles for the flexmaterial. Do I have something to modify ?

Ghostkeeper commented:
Speeds can be modified anywhere, so it's fine to have them in global or in the per-extruder quality profiles, whatever you like. The per-extruder quality profiles will override the global ones.

So I don't think you'd have to modify anything then.

Ghostkeeper commented:
Developers, see issue CURA-6631.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_0.6_abs_draft.inst.cfg:
```diff
@@ -6,7 +6,7 @@ definition = tizyx_evy
 setting_version = 7
 type = quality
-quality_type = draft
+quality_type = draft2
```
Forgot one here!

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_0.5_petg_draft.inst.cfg:
```diff
@@ -6,7 +6,7 @@ definition = tizyx_evy
 setting_version = 7
 type = quality
-quality_type = draft
+quality_type = draft3
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_0.6_petg_draft.inst.cfg:
```diff
@@ -6,7 +6,7 @@ definition = tizyx_evy
 setting_version = 7
 type = quality
-quality_type = draft
+quality_type = draft2
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_0.5_pla_draft.inst.cfg:
```diff
@@ -6,9 +6,10 @@ definition = tizyx_evy
 setting_version = 7
 type = quality
-quality_type = draft
+quality_type = draft3
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_0.6_pla_draft.inst.cfg:
```diff
@@ -6,7 +6,7 @@ definition = tizyx_evy
 setting_version = 7
 type = quality
-quality_type = draft
+quality_type = draft2
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.5_pla_bois_draft.inst.cfg:
```diff
@@ -0,0 +1,15 @@
+[metadata]
+setting_version = 7
+type = quality
+quality_type = draft3
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.6_pla_bois_draft.inst.cfg:
```diff
@@ -0,0 +1,15 @@
+[metadata]
+setting_version = 7
+type = quality
+quality_type = draft2
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/tizyx_evy_global_Draft2_Quality.inst.cfg:
```diff
@@ -0,0 +1,45 @@
+[metadata]
+setting_version = 7
+type = quality
+quality_type = draft2
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/tizyx_evy_global_Draft3_Quality.inst.cfg:
```diff
@@ -0,0 +1,45 @@
+[metadata]
+setting_version = 7
+type = quality
+quality_type = draft3
```
This quality type was renamed.

Ghostkeeper commented on resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_0.5_pla_draft.inst.cfg:
```diff
@@ -6,9 +6,10 @@ definition = tizyx_evy
 variant = 0.5mm
 
 [values]
+layer_height = 0.25
```
Layer height in a non-global profile here.

TiZYX commented:
Oops sorry, I forgot tu delete old files.

Ghostkeeper commented:
There's lots of merge conflicts now! Should we hold this PR as leading?

TiZYX commented:
What are the conflicts ?

Ghostkeeper commented:
Github indicates the conflicts on its web page as well. Here's what Git says about this merge:

```
Auto-merging resources/variants/tizyx_evy_dual_direct_drive.inst.cfg
CONFLICT (content): Merge conflict in resources/variants/tizyx_evy_dual_direct_drive.inst.cfg
Auto-merging resources/variants/tizyx_evy_dual_classic.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_k25/tizyx_k25_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/tizyx_evy_dual_global_Normal_Quality.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/tizyx_evy_dual_global_High_Quality.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/tizyx_evy_dual_global_Flex_Quality.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/tizyx_evy_dual_global_Flex_Only_Quality.inst.CFG
Auto-merging resources/quality/tizyx/tizyx_evy_dual/pla/tizyx_evy_dual_direct_drive_pla_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/pla/tizyx_evy_dual_direct_drive_pla_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/pla/tizyx_evy_dual_direct_drive_pla_flex.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/pla/tizyx_evy_dual_classic_pla_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/pla/tizyx_evy_dual_classic_pla_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/pla/tizyx_evy_dual_classic_pla_flex.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/petg/tizyx_evy_dual_direct_drive_petg_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/petg/tizyx_evy_dual_direct_drive_petg_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/petg/tizyx_evy_dual_classic_petg_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/petg/tizyx_evy_dual_classic_petg_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/abs/tizyx_evy_dual_direct_drive_abs_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/abs/tizyx_evy_dual_direct_drive_abs_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/abs/tizyx_evy_dual_classic_abs_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy_dual/abs/tizyx_evy_dual_classic_abs_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/tizyx_evy_global_Normal_Quality.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/tizyx_evy_global_Extra_Coarse_Quality.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/tizyx_evy_global_Draft_Quality.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/tizyx_evy_global_Coarse_Quality.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.8_pla_bois_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.8_pla_bois_extra_coarse.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.6_pla_bois_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.6_pla_bois_coarse.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.5_pla_bois_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.5_pla_bois_draft.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.4_pla_bois_normal.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.4_pla_bois_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.3_pla_bois_high.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/pla_bois/tizyx_evy_0.2_pla_bois_high.inst.cfg
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_1.0_pla_extra_coarse.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_1.0_pla_extra_coarse.inst.cfg left in tree.
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_1.0_pla_coarse.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_1.0_pla_coarse.inst.cfg left in tree.
Auto-merging resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_0.8_pla_extra_coarse.inst.cfg
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_0.8_pla_coarse.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_0.8_pla_coarse.inst.cfg left in tree.
Auto-merging resources/quality/tizyx/tizyx_evy/pla/tizyx_evy_0.6_pla_coarse.inst.cfg
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_1.0_petg_extra_coarse.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_1.0_petg_extra_coarse.inst.cfg left in tree.
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_1.0_petg_draft.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_1.0_petg_draft.inst.cfg left in tree.
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_1.0_petg_coarse.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_1.0_petg_coarse.inst.cfg left in tree.
Auto-merging resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_0.8_petg_extra_coarse.inst.cfg
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_0.6_petg_draft.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_0.6_petg_draft.inst.cfg left in tree.
Auto-merging resources/quality/tizyx/tizyx_evy/petg/tizyx_evy_0.6_petg_coarse.inst.cfg
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_1.0_abs_extra_coarse.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_1.0_abs_extra_coarse.inst.cfg left in tree.
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_1.0_abs_draft.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_1.0_abs_draft.inst.cfg left in tree.
CONFLICT (modify/delete): resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_1.0_abs_coarse.inst.cfg deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD. Version HEAD of resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_1.0_abs_coarse.inst.cfg left in tree.
Auto-merging resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_0.8_abs_extra_coarse.inst.cfg
Auto-merging resources/quality/tizyx/tizyx_evy/abs/tizyx_evy_0.6_abs_coarse.inst.cfg
Automatic merge failed; fix conflicts and then commit the result.
```

TiZYX commented:
I don't understand this : "deleted in f924ddcc1c7eef31d6884241c0d6ddcd3ec08102 and modified in HEAD"
I removed some files in my last commit on my branch. What do I have to do to delete it form HEAD too ?

Also, for the content problem of resources/variants/tizyx_evy_dual_direct_drive.inst.cfg, can't find what's wrong ?

Thanks for your help

Ghostkeeper commented:
> I don't understand this : "deleted in f924ddc and modified in HEAD"
I removed some files in my last commit on my branch. What do I have to do to delete it form HEAD too ?

This always confuses me as well. Basically this means that one branch deleted it (your branch) but at the same time the file was edited in another branch (master in this case). We've probably edited the file in master to increment the setting version for Cura 4.2, so the merge conflict would then be resolved by deleting it anyway.

I'm working on the Creawsomemod merge right now. Afterwards I can give some help.

TiZYX commented:
Ok, waiting for your help then, no idea how to fix those conflicts

Ghostkeeper commented:
I've fixed those commits very easily. In all cases it was about us modifying the `setting_version` for the new release. In the quality profiles it was about those files now being deleted (so just deleted them) and in the variant there were 2 new setting overrides, so I added those setting overrides together with the new `setting_version`.

I also fixed a typo in some setting names: `retracation` -> `retraction`. This makes me think you didn't test it very well, but that's on you guys!

Thanks for the changes. They have now been merged so they'll be published with Cura 4.2.

One further note that you might want to pay attention to: You've now added profiles for the generic materials (some of them at least). You've previously also added new materials of your own. There are no quality profiles for the TiZYX materials, even though they are selected by default.

TiZYX commented:
Well on the tests we did, our profiles worked with our TiZYX materials. Don't know how why it's not working anymore.

For example, we didn't modify the 'tizyx_evy_0.3_pla_high.inst.cfg' file, but the profile is not working. In the global profile 'tizyx_evy_global_High_Quality.inst.cfg' we just changed some values.

Can you tell me what to do to make our profiles working with our materials please ? :)

TiZYX commented:
Thanks a lot !

I tested again the profiles and they are working with our TiZYX material isn't it ?

Ghostkeeper commented:
Huh, yeah now it seems to work. Maybe I had something messed up with my branches then.
